includes/view-ukuupeople.php

```diff
@@ -0,0 +1,110 @@
+          echo "<tr><td><span class='contactdetailhead'>".__( 'Job Title', 'UkuuPeople' ) ."</span></td><td class='title-value'>". $custom['wpcf-ukuu-job-title'][0]. "</td></tr>";
+        }
+
+        if ( isset( $custom['wpcf-ukuu-twitter-handle'] ) ) {
+          echo "<tr><td><span class='contactdetailhead'>".__( 'Twitter Handle', 'UkuuPeople' ) ."</span></td><td class='title-value'><a href='http://twitter.com/". ltrim( $custom['wpcf-ukuu-twitter-handle'][0], '@') ."' target='_blank'>".$custom['wpcf-ukuu-twitter-handle'][0]."</a></td></tr>";
+        }
+        if ( isset( $custom['wpcf-ukuu-facebook-url'] ) ) {
+          echo "<tr><td><span class='contactdetailhead'>".__( 'Facebook URL', 'UkuuPeople' ) ."</span></td><td class='title-value'><a href='http://facebook.com/". $custom['wpcf-ukuu-facebook-url'][0] ."' target='_blank'>".$custom['wpcf-ukuu-facebook-url'][0]."</a></td></tr>";
+        }
+        if ( isset( $custom['wpcf-ukuu-date-of-birth'] ) && !empty( $custom['wpcf-ukuu-date-of-birth'][0] ) ) {
+          echo "<tr><td><span class='contactdetailhead'>".__( 'Date Of Birth', 'UkuuPeople' ) ."</span></td><td class='title-value'>". $custom['wpcf-ukuu-date-of-birth'][0] ."</td></tr>";
+        }
+
+        echo '</table></div></div><div id="second-sidebar-contact"><div id="addressblock">';
+        $html = "<table id='contactdetail-table'><tr><td><span class='contactdetailhead'>".__( 'Address', 'UkuuPeople' ) ."</span></td></tr>";
+        $streetaddress = $city = $country = $postalcode = $state ='';
+        if( isset ( $custom['wpcf-streetaddress'] ) ) {
+          $html .= "<tr><td>".$custom['wpcf-streetaddress'][0]."</td></tr>";
+        }
+        if( isset ( $custom['wpcf-streetaddress2'] ) ) {
+          $html .= "<tr><td>".$custom['wpcf-streetaddress2'][0]."</td></tr>";
+        }
+
+        $html .= "<tr><td></td></tr>";
+        $city = isset($custom['wpcf-city']) ? $custom['wpcf-city'][0] : '';
+        $state = isset($custom['wpcf-state']) ? $custom['wpcf-state'][0] : '';
+        $postalcode = isset($custom['wpcf-postalcode']) ? $custom['wpcf-postalcode'][0] : '';
+
+        $html .= "<tr><td>{$city}".(!empty($city) ? ', ' : '')."{$state} {$postalcode}</td></tr></table>";
+        echo $html.'</div><div id="tagblock">';
+        $html = "<table id='contactdetail-table'><tr><td><span class='contactdetailhead'>".__( 'Tags', 'UkuuPeople' ) ."</span></td></tr><tr>";
+        $tags = '';
+        if ( ! empty( $ctags ) ) {
+          $tags = implode( ', ' , $ctags );
+        }
+        $html .= "<td><p class='title-value'>$tags</p></td></tr></table>";
+        echo $html.'</div></div>';
+        echo '</div>'; //post-body-content closed
+        echo '</div>'; //post-body-content closed
+        //~ do_action( 'tab_info' , $people_object ,$type[0]->slug );
+        echo'<a class="button button-primary" href="'.admin_url('edit.php?post_type=wp-type-contacts').'">Back</a>';
```

Comment:
There must have to be space between echo and '.. instead of single quotes here, double quotes should have been used.